AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -51,6 +51,8 @@ gaugau.futabanet.jp,newscientist.com,jiji.com,edp24.co.uk,technologyreview.com,n
 !
 ! SECTION: Subscriptions - Regular rules
 !
+osce.org##aside.osce-box.osce-box-style-bar.osce-box-social_sharing_and_contextual_1
```

Comment:
Two things.
First - currently it hides whole panel, not only share element, but if I'm not wrong, it looks like that there is a button to change language which should not be blocked.
So please change it to hide only `pane-social-sharing` element.

Second, please move it to `! NOTE: Specific rules` in AdGuard Social Media filter https://github.com/AdguardTeam/AdguardFilters/blob/master/SocialFilter/sections/specific.txt#L83-L86